index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/9858e22e8a.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <script src="main.js" defer></script>
+    <title>Project</title>
+</head>
+<body>
+    <section class="main-container">
+        <img class="toggle" src="images/toggle.png" alt="toggle-bar">
+        <ul class="first-header">
+            <li class="top-menu"><img class="facebook" src="images/facebook-icon.jpg" alt="facebook icon"></li>
+            <li class="top-menu"><img class="twitter" src="images/twitter-icon.png" alt="facebook icon"></li>
+            <li class="top-menu">English</li>
+            <li class="top-menu">My Page</li>
+            <li class="top-menu">Layout</li>
+        </ul>
+        <nav class="second-nav hide-menu">
+            
+            <div class="logo__ul">
+                <div class="close">&times;</div>
+                <p class="logo">Logo</p>
+                <ul class="displayf">
+                    <li>About</li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponsor</li>
+                    <li>News</li>
+                </ul>
+            </div>
+            <div class="campcc">
+                <p class="camp">cc Campaign</p>
+            </div>
+        </nav>
+        <section class="sections">
+            <section class="intro-section">
+                <p>"Hello! Sharing world"</p>
+                <h2>BUILDING A BLACKGIRLS TECH COMMUNITY 2022</h2>
+                <div class="box">
+                    <p>Creating a tech community to empower young black women to surface the tech field. Black girls stand.
+                    </p>    
+                </div>
+                <div class="date">
+                    <h3>2022.01.03(MON) ~ 8(SAT)</h3>
+                    <p>@ S.S.A.M.B empowerment of West and Central Africa</p>
+                </div>
+            </section>
```

Comment:
- [ ] I think this section's elements are well aligned. You really did a great job! 👏🏻 . However, in reference to the behance design it would be nice if you added a background image with a faint grey-tint.

Reply:
RESOLVED
Background Image added to the section and blended.